packages/client/webapp/src/components/Sidebar.vue

```diff
@@ -158,8 +162,7 @@ aside {
 	display: flex;
 	flex-direction: column;
 	justify-content: space-between;
-	padding: 0.5rem;
-	width: 15rem;
+	width: max(15em, 20vw);
```

Comment:
I don't know if we had discussed this, but are we using `rem` or we are using `em`? Would prefer if we stick to one system rather than mixing both. Slight personal preference for `rem`.

Reply:
They are not the same, I don't see how can this be an issue?

Reply:
Exactly, there's nothing technically wrong here, they are both relative measurements but with different basis. 

So when you mix and match it is like you are working with two different coordinate systems. Potentially it can be confusing, especially if components are meant to embed into others. Imaging the inner-most `div` is a reusable component.

```
<html>
  <head>
  </head>
  <body>
    <div style="font-size: 1em"> Hi
      <div style="font-size: 2em"> Hi
        <div style="font-size: 2em"> Hi
        </div>
      </div>
    <div>
    <hr>
    <div style="font-size: 1rem"> Hi
      <div style="font-size: 2rem"> Hi
        <div style="font-size: 2rem"> Hi
        </div>
      </div>
    <div>
    <hr>
    <div style="font-size: 1em"> Hi
      <div style="font-size: 2em"> Hi
        <div style="font-size: 2rem"> Hi
        </div>
      </div>
    <div>
  </body>
</html>
```

<img width="126" alt="image" src="https://user-images.githubusercontent.com/1220927/195406142-ae540988-d2c9-45ec-8e46-42b641ff639b.png">

Reply:
I agree with you, but we are using the font-system from the Uncharted design tokens, and we scope our css as much as possible.

After it's important to let the design be responsibe to a change a font-size within a component or it's parent.
This is why I use `em` when its a behaviour I'm expecting.

Reply:
But we are using `rem` in other places.
The font variables are also specified in `rem` so they are not responsive per se, and  the scoping will not help in this case I believe, because it searches the nearest parent-node regardless.

Reply:
Fine, I'll stop using `em` and choose a unit tied to the parent component as source of responsiveness instead of font